elephant/causality/granger.py

```diff
@@ -596,3 +782,168 @@ def conditional_granger(signals, max_order, information_criterion='aic'):
                                                  est_sig_figures)
 
     return conditional_causality_xy_z_round
+
+
+def pairwise_spectral_granger(signal_i, signal_j, fs=1, nw=4, num_tapers=None,
+                              peak_resolution=None, n_segments=1,
+                              len_segment=None, frequency_resolution=None,
+                              overlap=0.5, num_iterations=300,
+                              term_crit=1e-12):
+    r"""Determine spectral Granger Causality of two signals.
+
+    The spectral Granger Causality is obtained through the following steps:
+
+    1. Determine the cross spectrum of the two signals by applying
+       :func:`segmented_multitaper_cross_spectrum` to the joint signal. See the
+       documentation of this function for the parameter hierarchy of the
+       used for the estimation of the cross spectrum.
+
+    2. Calculate the spectral factorization of the cross spectrum decomposing
+       the very same thing into the covariance matrix and the transfer
+       function.
+
+    3. Calculate the directional and instantaneous spectral Granger Causality
+       from the power spectra, the transfer function and the covariance matrix.
+
```

Comment:
Reference to relevant literature would be preferable.

Reply:
Addressed in 0f6ea81.